html/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>myhomepage</title>
+  </head>
+  <body>
+    <div>
+      <h1>Tobias Schweizer</h1> 
+      <img 
+          style="float: left;"
+          src="Tobias.jpeg"
+          alt="my picture"
+          width="100"
+        />
+    
+
+      <nav id="nav">
+  <br>
+  <center>
+        <p> <center> </center><a  href="#contact">Contact</a></p>
```

Comment:
this center is unnecessary :)
Also you should wrap the a tag in an list item instead of a p
Then there should also be a ul wrapping around them of course :)